### PR TITLE
Added components for events page

### LINKED ISSUE #3: Create Events Page (Tech Team)
Create events page that will list all the upcoming and previous events.

### PR DESCRIPTION
**Pull request by ameyaaade**
Added components for events page
handled desktop as well as mobile view

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -19,6 +20,8 @@ function App() {
           <Route path='/register' element={<RegistrationPage />}></Route>
           <Route path='/about' element={<AboutPage />}></Route>
           <Route path='/admin' element={<AdminPage />}></Route>
+          <Route path='/events' element={<EventsPage />}></Route>
+          <Route path='/*' element={<h1>404 Page not found!</h1>}></Route>
```
Comment (rajcrk) [inline src/App.js]: Nice! This is good thinking. We don't have this now! 🔥

File: src/App.js [hunk 2]
```diff
@@ -19,6 +20,8 @@ function App() {
           <Route path='/register' element={<RegistrationPage />}></Route>
           <Route path='/about' element={<AboutPage />}></Route>
           <Route path='/admin' element={<AdminPage />}></Route>
+          <Route path='/events' element={<EventsPage />}></Route>
```
Comment (rajcrk) [inline src/App.js:24]: Let's add a link in the `Header.component.js` for events route.

File: src/pages/EventsPage/EventsPage.page.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      subHeading: "this is sub heading",
+      location: "Toldo health center",
+      time: "22 Apr 2023",
+      describe:
+        "This is a description of an event, This is a description of an event, This is a description of an event,This is a description of an event, This is a description of an event",
+    },
+  ];
+  return (
+    <div className="eventsPageWrapper">
+      <h1 className="title"> Upcomin Events</h1>
```
Comment (rajcrk) [inline src/pages/EventsPage/EventsPage.page.js]: Let's try to use carbon design CSS so the website is consistent.

Similar to what we did here: 

```html
<p className='heading-02 member-heading'>
       Meet the board members and <u>connect with us!</u>
</p>
```

The list of typeset is here: https://carbondesignsystem.com/guidelines/typography/type-sets/

Comment (rajcrk) [inline src/pages/EventsPage/EventsPage.page.js]: Upcoming Events (Typo)

File: src/pages/ErrorPage/ErrorPage.page.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```
Comment (rajcrk) [inline src/pages/ErrorPage/ErrorPage.page.js]: Screen Shot 2023-04-28 at 7 37 31 PM

I'm unable to see 404 Error Message.

Let's add top margin to the div to make It visible and make sure that the typography is from carbon  design. Make use of the grid system from the UI library. Its faster that way.

https://carbondesignsystem.com/guidelines/2x-grid/overview/

File: package-lock.json
Comment (rajcrk) [inline package-lock.json:1]: There is no need to commit lock file. If there are no changes to package.json file.

File: src/App.js (hunk 1 above)
Comment (ameyaaade) [reply src/App.js]: Thanks!

File: src/App.js (hunk 2 above)
Comment (ameyaaade) [reply src/App.js:24]: Updated.

File: src/pages/EventsPage/EventsPage.page.js (hunk 3 above)
Comment (ameyaaade) [reply src/pages/EventsPage/EventsPage.page.js]: Updated

Comment (ameyaaade) [reply src/pages/EventsPage/EventsPage.page.js]: fixed

File: src/pages/ErrorPage/ErrorPage.page.js (hunk 4 above)
Comment (ameyaaade) [reply src/pages/ErrorPage/ErrorPage.page.js]: Used custom styles 
to make it less complex

File: package-lock.json
Comment (ameyaaade) [reply package-lock.json:1]: Keeping it as discussed .

**Comment by rajcrk** [comment]
Let's update the previous events array of objects with actual data as discussed.

**Pull request merged**